### PR TITLE
changed way to compute face center and edge midpoint

### PR DESCRIPTION
**Pull request by maxcapodi78**

### CODE REVIEW

File: pyaedt/modeler/cad/elements3d.py [hunk 1]
```diff
@@ -342,14 +342,14 @@ def midpoint(self):
...
-        else:
-            return [float(i) for i in self.oeditor.GetEdgePositionAtNormalizedParameter(self.id, 0)]
+        return [float(i) for i in self.oeditor.GetEdgePositionAtNormalizedParameter(self.id, 0.5)]
+        # if len(self.vertices) == 2:
+        #     midpoint = GeometryOperators.get_mid_point(self.vertices[0].position, self.vertices[1].position)
+        #     return list(midpoint)
+        # elif len(self.vertices) == 1:
+        #     return self.vertices[0].position
+        # else:
+        #     return [float(i) for i in self.oeditor.GetEdgePositionAtNormalizedParameter(self.id, 0)]
```
Comment (MaxJPRey) [inline pyaedt/modeler/cad/elements3d.py:352]:
```suggestion
```

**Review by Alberto-DM — COMMENTED** [review]
LGTM

**Review by Alberto-DM — APPROVED** [review]
LGTM

**Comment by mrjoesnow** [comment]
Hi,

This new method causes Ansys to spew lots of warnings in the message log every time center is called (which is alot of places)

[this](https://github.com/ansys/pyaedt/blob/1b85aca1e9d67afd3c929b46cddc8cd3ab0ba0fd/pyaedt/modeler/cad/elements3d.py#L600C59-L600C59
) is the offending line

and causes 

```
    [error] Script macro error: Failed to compute face center as the face is not planar.  (2:47:41 PM  Jun 23, 2023)
    [error] Script macro error: Failed to compute face center as the face is not planar.  (2:47:41 PM  Jun 23, 2023)
    [error] Script macro error: Failed to compute face center as the face is not planar.  (2:47:41 PM  Jun 23, 2023)
    [error] Script macro error: Failed to compute face center as the face is not planar.  (2:47:41 PM  Jun 23, 2023)
```

**Pull request merged**